src/pybind/mgr/dashboard/frontend/src/app/shared/components/custom-login-banner/custom-login-banner.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+
+import _ from 'lodash';
+
+import { CustomLoginBannerService } from '~/app/shared/api/custom-login-banner.service';
+
+@Component({
+  selector: 'cd-custom-login-banner',
+  templateUrl: './custom-login-banner.component.html',
+  styleUrls: ['./custom-login-banner.component.scss']
+})
+export class CustomLoginBannerComponent implements OnInit {
+  customText: string;
+  constructor(private customLoginBannerService: CustomLoginBannerService) {
+    this.customText = null;
+  }
+
+  ngOnInit(): void {
+    this.customLoginBannerService.getBannerText().subscribe((response: any) => {
+      if (_.isString(response)) {
```

Comment:
?
```suggestion
      if (response) {
```

Reply:
ah, probably was testing something and forgot to change it back, thanks!.